### DISCUSSION TITLE
Latest 0.62.2+ releases are missing windows installer (.msi)

### DISCUSSION THREAD

**Discussion by Dragon31337** [mesonbuild/meson#10467] [General]
Looks like 0.62.2 release does not have an msi file.

**Comment by Dragon31337** [comment]
All newer versions are missing windows installers as well.

**Comment by Dragon31337** [comment]
Pinging once again.

**Comment by eli-schwartz** [comment]
We tried to build one for the 0.63.0 release but could not do so, see https://github.com/pyinstaller/pyinstaller/issues/6929

**Reply by eli-schwartz** [reply]
This is now fixed and released upstream, so we will try to publish this again, tomorrow.

**Reply by nyorain** [reply]
Thanks for looking into it. Any news on this?

**Comment by jpakkane** [comment]
I just uploaded the latest installer so this should be fixed now.

**Comment by Dragon31337** [comment]
0.64 RC builds are missing msi files again

**Reply by eli-schwartz** [reply]
AFAICT we never uploaded those for release candidates, only stable releases.

@jpakkane what do you think?
